website/docs/r/org_vdc_template_instance.html.markdown

```diff
@@ -0,0 +1,136 @@
+    name    = "*"
+    default = true
+    limit   = 1024
+  }
+
+  network_pool_id = data.vcd_network_pool.np1.id
+
+  readable_by_org_ids = [
+    data.vcd_org.org.id
+  ]
+}
+
+resource "vcd_org_vdc_template_instance" "my_instance" {
+  org_vdc_template_id = vcd_org_vdc_template.tmpl.id
+  name                = "myInstantiatedVdc"
+  description         = "A new VDC"
+  org_id              = data.vcd_org.org.id
+
+  delete_instantiated_vdc_on_removal = true
+  delete_force                       = true
+  delete_recursive                   = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org_vdc_template_id` - (Required) The ID of the VDC Template to instantiate
+* `name` - (Required) Name to give to the instantiated Organization VDC
+* `description` - (Optional) Description of the instantiated Organization VDC
+* `org_id` - (Required) ID of the Organization where the VDC will be instantiated
+* `delete_instantiated_vdc_on_removal` - (Required) If this flag is set to `true`, removing this resource will attempt to delete the instantiated VDC
+* `delete_force` - (Optional) Defaults to `false`. If this flag is set to `true`, it forcefully deletes the VDC, only when `delete_instantiated_vdc_on_removal=true`
+* `delete_recursive` - (Optional) Defaults to `false`. If this flag is set to `true`, it recursively deletes the VDC, only when `delete_instantiated_vdc_on_removal=true`
+
+## Attribute Reference
+
+There are no read-only attributes. However, after the `vcd_org_vdc_template_instance` resource is created successfully,
+the identifier of the new VDC is saved in the Terraform state, as the `id` of the `vcd_org_vdc_template_instance` resource
+(example: `vcd_org_vdc_template_instance.my_instance.id`).
+
+## How to manage the VDC instantiated from a template using Terraform
+
+If users want to modify the new instantiated VDC, they can [import](/providers/vmware/vcd/latest/docs/guides/importing_resources#semi-automated-import-terraform-v15) it.
+In the same `.tf` file (once the VDC has been instantiated), or in a new one, we can place the following snippet: 
+
+```hcl
+import {
+  to = vcd_org_vdc.imported
+  id = "my_org.myInstantiatedVdc" # Using the same names from the example
+}
+```
+
+Note that this importing mechanism still does not support `${}` placeholders, so the Organization and VDC name must be explicitly
+written. When running the `terraform plan -generate-config-out=generated_resources.tf`, Terraform will generate the new file
+`generated_resources.tf` with the instantiated VDC code.
+
+With a subsequent `terraform apply`, the instantiated VDC will be managed by Terraform as a normal `vcd_org_vdc` resource.
+
+-> After importing, bear in mind that `vcd_org_vdc` will have the arguments `delete_force` and `delete_recursive` set to `false`.
+They should be modified accordingly.
+
+If the `delete_instantiated_vdc_on_removal` argument of `vcd_org_vdc_template_instance` is set to `true`, removing
```

Comment:
I think this should be a warning. Also, I think it could be listed above or separately from the importing paragraphs, because it is not required to do import in order to delete it.

Actually, the whole deletion topic deserves a separate section :) It could warn and describe all three params.

Reply:
Done, moved to its own section